data/mods/Medieval_Stuff/tools.json

```diff
@@ -0,0 +1,543 @@
+    "description": "This is a curved sword associated with various middle eastern and central asian countries. Designed for slashing, and quite deadly against unarmored targets.",
+    "price": 93000,
+    "material": "iron",
+    "techniques": ["RAPID", "WBLOCK_1"],
+    "flags" : ["SLICE", "DURABLE_MELEE", "SHEATH_SWORD"],
+    "weight": 1133,
+    "volume": 8,
+    "bashing": 6,
+    "cutting": 30,
+    "to_hit": 1,
+    "max_charges": 0,
+    "initial_charges": 0,
+    "charges_per_use": 0,
+    "turns_per_charge": 0,
+    "ammo": "NULL",
+    "revert_to": "null",
+    "use_action": { "type": "knife" },
+    "category" : "weapons",
+    "qualities": [["CUT", 1], ["BUTCHER", 8]]
+  },
+    {
+        "type":"TOOL",
+        "id": "sword_xiphos",
+        "symbol" : "!",
+        "color": "yellow",
+        "name": "xiphos",
+        "name_plural": "xiphoses",
+        "description": "A bronze sword of ancient Greek origin, wielded as a sidearm to the dory spear.",
+        "material": "bronze",
+        "price": 12000,
+        "volume": 8,
+        "weight": 800,
+        "bashing": 4,
+        "cutting": 16,
+        "to_hit": 2,
+        "max_charges": 0,
+        "initial_charges": 0,
+        "charges_per_use": 0,
+        "turns_per_charge": 0,
+        "ammo": "NULL",
+        "revert_to": "null",
+        "category": "weapons",
+        "flags": ["SHEATH_SWORD", "DURABLE_MELEE"],
+        "techniques": ["WBLOCK_1"],
+        "use_action": { "type": "knife" },
+        "qualities": [["CUT", 1], ["BUTCHER", 8]]
+    },
+  {
+    "id": "khopesh",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "yellow",
+    "name": "khopesh",
+    "name_plural": "khopeshes",
+    "description": "This ancient bronze weapon features a curved, sickle-like blade sharpened on the outside edge. Associated with the New Kingdom period of ancient Egypt, it was designed mainly to hack through the light armor common to the region.",
+    "price": 12000,
+    "material": "bronze",
+    "techniques": ["WBLOCK_1", , "DEF_DISARM"],
+    "weight": 750,
+    "volume": 7,
+    "bashing": 6,
+    "cutting": 18,
+    "flags" : ["CHOP", "DURABLE_MELEE"],
+    "to_hit": 1,
+    "max_charges": 0,
+    "initial_charges": 0,
+    "charges_per_use": 0,
+    "turns_per_charge": 0,
+    "ammo": "NULL",
+    "revert_to": "null",
+    "category": "weapons",
+    "use_action": { "type": "knife" },
+    "qualities": [["CUT", 1], ["BUTCHER", 8]]
+  },
+  {
+    "id": "jian",
+    "type": "TOOL",
+    "symbol": "/",
+    "color": "light_gray",
+    "name": "jian",
+    "name_plural": "jian",
+    "description": "This is an ancient Chinese doubled-edged straight sword, with an ornate guard and a tassel attatched to the pommel.  One of the four major weapons in folklore, alongside the dao sabre, qiang spear, and gun staff.",
+    "price": 100000,
+    "material": "steel",
+    "flags" : ["CHOP", "DURABLE_MELEE", "SHEATH_SWORD"],
+    "techniques": ["RAPID", "PRECISE", "WBLOCK_1"],
```

Comment:
Same here. Rapid for an 8 volume weapon is rather unrealistic.
Not sure about precise. I think bumping to-hit would fit it better, if it is easy to strike precisely with.